test/clj/villagebook/item/models_test.clj

```diff
@@ -0,0 +1,26 @@
+(ns villagebook.item.models-test
+  (:require [villagebook.fixtures :refer [setup-once wrap-transaction]]
+            [villagebook.field.db :as field-db]
+            [villagebook.user.db :as user-db]
+            [villagebook.organisation.models :as org-models]
+            [villagebook.category.db :as category-db]
+            [villagebook.item.models :as models]
+            [villagebook.factory :as factory]
+            [clojure.test :refer :all]))
+
+(use-fixtures :once setup-once)
+(use-fixtures :each wrap-transaction)
+
+(deftest create-tests
+  (testing "Should create an item with given fields"
+    (let [user-id     (:id (user-db/create! factory/user1))
+          org         (:success (org-models/create! factory/organisation user-id))
```

Comment:
Any reason for creating `org` via its model, but others via their db namespaces?

Reply:
Creating the org through the model fn takes care of adding the `owner` permission to the user who created it. The db functions are used otherwise because they are simpler (lesser checks, no success/error map).